en/lessons/cleaning-data-with-openrefine.md

```diff
@@ -292,7 +292,7 @@ cells**, **Join multi-valued cells**, **OK**. Choose the pipe character (|) as a
 separator. The rows now look like before, with a multi-valued Categories
 field.
 
-### Applying ad-hoc transformations through the use of regular expressions
+### Applying ad-hoc transformations through the use of GREL expressions
```

Comment:
This section does not use regular expressions at all, only GREL expressions, so I think it's worth avoiding this confusion.